conda/cli/main_config.py

```diff
@@ -365,6 +365,29 @@ def execute_config(args, parser):
                                 (item, key))
         rc_config[key] = [i for i in rc_config[key] if i != item]
 
+    # Clear
+    for key, in args.clear:
+        key, subkey = key.split('.', 1) if '.' in key else (key, None)
```

Comment:
Is this `key.subkey` notation supported in other commands too?

Reply:
yup- this is supported for prepend, append, and add operations:
https://github.com/conda/conda/blob/448fc905536ffd5a39781aeb50076c7831e302dd/conda/cli/main_config.py#L311

does it make sense to also support for clear, for consistency? or would starting with only supporting primary keys cover most valid use cases of clearing out an config array?